---
layout: default
title: Projects
---

<div class="col-sm-12 col-md-11 col-lg-10 col-xxl-9 align-middle mx-auto d-block">
    <div class="blurb">
        <h1>Projects</h1>

        <button type="button" class="collapsible">Phage-Host Coevolution</button>
        <div class="content">
            <br>
            <div class="row container">
                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                    <img alt="phage_ecoli" src="https://media.newyorker.com/photos/590966916552fa0be682efe5/master/pass/Twilley-phages.jpg" class="img-fluid rounded align-middle mx-auto d-block" style="max-width: 50%;">
                </div>
                <figcaption class="figure-caption text-center">Bacteriophages infecting <i>E. coli</i>. Credits Eye of Science.</figcaption>
                
            </div>
            <br>
            <div class="col-md-11 col-lg-10 col-xl-10 col-xxl-9 align-middle mx-auto d-block">
                <p>
                    Bacterial hosts engage in antagonistic interactions with the lytic phages that decrease the survival of these hosts.
                    To limit these negative interactions, bacteria must evolve ways to defend themselves and resist phage infection. 
                    This resistance can manifest itself in several forms: modifying the receptors phages need to attach, destroying invasive
                    DNA or RNA once inside the host (restriction modification, CRISPR-Cas), or even early cell death to prevent larger outbreaks (abortive infection).
                </p>
                <p>
                    These mechanisms are only a small portion of the vast array of ways bacteria can defend themselves against phage. Why do so many mechanisms exist? 
                    One potential explanation is the other side of this picture: phage evolution. Phage mutations that can overcome the defense mechanisms of 
                    their bacterial hosts confer advantages to these new mutants. These new mutations can lead to new responses in the hosts to combat these new and 
                    improved phage, thus starting a cycle of coevolution. 
                </p>
                <p>
                    This coevolution is the focus of my eco-evolutionary simulation model currently called <a href="https://github.com/fordfishman/CommunityImmunity">CommunityImmunity</a>. 
                    In this model, host and phage strains are allowed to mutate and create new strains that may be phage-resistant or may have a larger host range. The demographic
                    information of all strains is tracked and partially determines the frequency of mutation. I am currently using this model to explore the impact of increasing
                    host and phage mutation rate on the outcome of each simulation and the mode of coevolution. 
                </p>
            </div>
        </div>

        <button type="button" class="collapsible">Constraints on Microbial Diversity</button>
        <div class="content">
            <p>
                Measuring microbial diversity through genomics is difficult due to the sheer number of 
                individuals and species in a given environment, as well as genetic heterogeneity. To 
                study the ecology of microbial communities, it is common practice to use marker genes, 
                such as sections of the 16S rRNA gene, as a sort of proxy of actual diversity. I am broadly interested in the 
                evolution of 16S and using it to measure global microbial diversity. I am also interested in how
                diversity at this marker locus corresponds to actual functional and species level diversity in bacteria.
            </p>
        </div>

        <button type="button" class="collapsible">Mutualism and Dormancy</button>
        <div class="content">
            <br>
            <div class="row container">
                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                    <img alt="phage_ecoli" src="/pics/cellfield.gif" class="img-fluid rounded align-middle mx-auto d-block" style="max-width: 70%;">
                </div>
                <figcaption class="figure-caption">
                    Simulation of the spatial model showing dynamics over 1000 timesteps of A (blue) and B (green) cooperators, 
                    as well as dormant A (red) and B (yellow) cells, on a 2D plane. 
                    Black edges are present to distinguish between cells.
                </figcaption>
                
            </div>
            <br>
            <div class="col-md-11 col-lg-10 col-xl-10 col-xxl-9 align-middle mx-auto d-block">
                <p>
                    Interactions between species or strains can be considered mutualistic when each species benefits from the presence of the other. 
                    Mutualisms are very common in nature, such as the relationship between plants and their insect pollinator. 
                    In bacteria, one form of mutualism occurs when different strains produce resources that can used by other strains. 
                    However, while receiving benefits from a mutualistic partner can increase growth of an organism, producing the benefits 
                    for that partner might be costly. These costs may inhibit the evolution of these mutualisms or perhaps lead to cheating, 
                    where some individuals receive the benefits of the mutualism without producing any costly resources themselves.
                </p>
                <p>
                    Dormancy, where certain individuals become inactive for some time, may help to stabilize these relationships. 
                    For instance, while the costs of mutualism may lead to a disadvantage when resources are abundant, it may be beneficial 
                    when resources are scarce. A seed bank of dormant cells may allow mutualistic cells to persist when selection favors 
                    cheaters until periods where it favors cooperation. Additionally, environmental stochasticity can lead to extinction of 
                    one of the two partners by chance. A seed bank can drastically reduce this probability by maintaining a seed bank of both partners. 
                </p>
                <p> 
                    To model situations where dormancy can help maintain cooperation, I developed two individual-based models (IBMs): 
                    one non-spatial, and one spatial. Both implement cells as the primary individuals, or agents, 
                    and these cells can be of two types existing in a single population. They both share a common resource and 
                    produce secondary resources that can only be used by the other cell type. Both implement stochastic processes 
                    and allow individuals to make decisions based upon their current attributes. For more information about implementation, 
                    please go to the project <a href="https://github.com/fordfishman/mutual-dorm">GitHub repo</a>. 
                </p>
                <p> 
                    These models have shown that when two mutualistic species are capable of dormancy, they can coexist in the long-term 
                    despite extreme resource limitation. Normally, without dormancy, neither species can survive, and without cooperation, 
                    one strain will go extinct. With both dormancy and mutualism, both are maintained. In the future, both models will 
                    require thorough parameter sweeps to identify precise parameters where dormancy increases the probability of coexistence.
                    
                </p>
            </div>
            <br>
            <div class="row container">
                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
                    <img alt="phage_ecoli" src="/pics/celldensities.png" class="img-fluid rounded align-middle mx-auto d-block" style="max-width: 70%;">
                </div>
                <figcaption class="figure-caption">
                    Simulation of energy limitation showing sustained coexistence with mutualism (trait = 0.1) and stochastic dormancy enabled.
                </figcaption>                
            </div>
            <br>
        </div>

        <script>
            var coll = document.getElementsByClassName("collapsible");
            var i;

            for (i = 0; i < coll.length; i++) {
            coll[i].addEventListener("click", function() {
                this.classList.toggle("active");
                var content = this.nextElementSibling;
                if (content.style.maxHeight){
                    content.style.maxHeight = null;
                } else {
                    content.style.maxHeight = content.scrollHeight + "px";
                } 
            });
            }
        </script>

        <!-- <h3>Marker Genes in Microbial Evolution and Ecology</h3>
        <p>
            Measuring microbial diversity through genomics is difficult due to the sheer number of 
            individuals and species in a given environment, as well as genetic heterogeneity. To 
            study the ecology of microbial communities, it is common practice to use marker genes, 
            such as sections of the 16S rRNA gene, as a sort of proxy of actual diversity.  
        </p> -->
    </div>
</div>